src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -651,11 +659,49 @@ private unsafe void WriteSnapshotToFile(ClusterOperationContext context, Stream
                                                 var currentTreeValueKey = treeIterator.CurrentKey;
                                                 binaryWriter.Write(currentTreeValueKey.Size);
                                                 copier.Copy(currentTreeValueKey.Content.Ptr, currentTreeValueKey.Size);
-                                                var reader = treeIterator.CreateReaderForCurrent();
-                                                binaryWriter.Write(reader.Length);
-                                                copier.Copy(reader.Base, reader.Length);
-
-                                                totalSizeInBytes += sizeof(long) + currentTreeValueKey.Size + reader.Length;
```

Comment:
previously, we had `sizeof(long)` here, but then `sizeof(int`) ??

Reply:
`totalSizeInBytes` is only for logging. 
the `sizeof(long)` was to account the sending of `inaryWriter.Write(reader.Length)` but `reader.Length` is an `int`

Reply:
Isn't that a change in the size of the data between versions?

Reply:
we don't care, we use the `totalSizeInBytes` only in the `Log.Info`